aws-codeartifact-domain/aws-codeartifact-domain.json

```
@@ -39,12 +39,9 @@
       ],
       "type": "string"
     },
-    "PolicyDocument": {
+    "PermissionsPolicyDocument": {
       "description": "The access control resource policy on the provided domain.",
-      "type": [
-        "object",
-        "string"
-      ],
+      "type": "object",
       "minLength": 2,
       "maxLength": 5120
```

[`minLength`/`maxLength` is specifically for `string` types](https://json-schema.org/understanding-json-schema/reference/string.html#length)
[issue to catch these automatically](https://github.com/aws-cloudformation/cloudformation-cli/issues/414)
```suggestion
      "type": "object",
```